core/src/main/scala/kafka/server/KafkaBroker.scala

```diff
@@ -121,6 +125,19 @@ trait KafkaBroker extends Logging with KafkaMetricsGroup {
   def currentState(): BrokerState
   def clusterId(): String
 
+  // methods required for legacy/kip500-agnostic dynamic configuration
+  def getConfig(): KafkaConfig
```

Comment:
In Kafka we don't preface accessors with "get"

So this should be config(), authorizer(), etc. etc.

Reply:
> In Kafka we don't preface accessors with "get"
So this should be config(), authorizer(), etc. etc.

I changed them all to have parameterless definitions (e.g. config, authorizer, etc. rather than config(), authorizer(), etc.) since that seems to be appropriate when the methods have no side-effects (which these do not), and it also means less change throughout (no need to add all those empty parens all over the place).